diff --git a/node_modules/@codetrix-studio/capacitor-google-auth/android/.classpath b/node_modules/@codetrix-studio/capacitor-google-auth/android/.classpath
new file mode 100644
index 0000000..bbe97e5
--- /dev/null
+++ b/node_modules/@codetrix-studio/capacitor-google-auth/android/.classpath
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-17/"/>
+	<classpathentry kind="con" path="org.eclipse.buildship.core.gradleclasspathcontainer"/>
+	<classpathentry kind="output" path="bin/default"/>
+</classpath>
diff --git a/node_modules/@codetrix-studio/capacitor-google-auth/android/.project b/node_modules/@codetrix-studio/capacitor-google-auth/android/.project
new file mode 100644
index 0000000..80b0af4
--- /dev/null
+++ b/node_modules/@codetrix-studio/capacitor-google-auth/android/.project
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>codetrix-studio-capacitor-google-auth</name>
+	<comment>Project codetrix-studio-capacitor-google-auth created by Buildship.</comment>
+	<projects>
+	</projects>
+	<buildSpec>
+		<buildCommand>
+			<name>org.eclipse.jdt.core.javabuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+		<buildCommand>
+			<name>org.eclipse.buildship.core.gradleprojectbuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+	</buildSpec>
+	<natures>
+		<nature>org.eclipse.jdt.core.javanature</nature>
+		<nature>org.eclipse.buildship.core.gradleprojectnature</nature>
+	</natures>
+</projectDescription>
diff --git a/node_modules/@codetrix-studio/capacitor-google-auth/android/.settings/org.eclipse.buildship.core.prefs b/node_modules/@codetrix-studio/capacitor-google-auth/android/.settings/org.eclipse.buildship.core.prefs
new file mode 100644
index 0000000..68c9fab
--- /dev/null
+++ b/node_modules/@codetrix-studio/capacitor-google-auth/android/.settings/org.eclipse.buildship.core.prefs
@@ -0,0 +1,2 @@
+connection.project.dir=../../../../android
+eclipse.preferences.version=1
diff --git a/node_modules/@codetrix-studio/capacitor-google-auth/android/build.gradle b/node_modules/@codetrix-studio/capacitor-google-auth/android/build.gradle
index 4bb66ea..4f311a0 100644
--- a/node_modules/@codetrix-studio/capacitor-google-auth/android/build.gradle
+++ b/node_modules/@codetrix-studio/capacitor-google-auth/android/build.gradle
@@ -7,7 +7,7 @@ ext {
 
 buildscript {
     repositories {
-        jcenter()
+        mavenCentral()
         google()
     }
     dependencies {
@@ -45,7 +45,6 @@ android {
 
 repositories {
     google()
-    jcenter()
     mavenCentral()
 }
 
